docker-wrappers/AllPairs/metadata.json

```diff
@@ -0,0 +1,4 @@
+{
+    "dockerName": "allpairs",
```

Comment:
`dockerName` feels slightly off but I'm not sure what else to call this. `image`? `imageName`?

Reply:
`imageName` sounds good. 👍